### ISSUE TITLE
App catalog broken?

### ISSUE THREAD

**Issue by skddc** [unhosted/store#2]
This is all I can see on apps.unhosted.org currently:

![screenshot from 2015-07-27 12-06-16](https://cloud.githubusercontent.com/assets/842/8903380/f621a408-3457-11e5-903b-6fa241431e59.png)

When I click the button, the "loading" disappears and it says done. Clicking it again toggles back to exactly this state. Is it broken or just unclear how to find/add apps/channels?

**Comment by skddc** [comment]
@michielbdejong Could you enlighten me plz? :)

I'm renewing the cert for this domain today, because it's expiring, but it's a bit disappointing to do that for something that doesn't actually work.

**Comment by michielbdejong** [comment]
Right, it's because it's still using https://store.unhosted.org/defaultApps.json instead of https://apps.unhosted.org/defaultApps.json as the default app channel. If you change the App channel URL to apps. and then click 'Update' again, it works.

**Comment by raucao** [comment]
Thanks. Still fails due to the expired cert. Can you add that CNAME record?

**Comment by gunar** [comment]
Commit 306dc6c fixes it but the live site is still outdated. How can I help?

**Comment by michielbdejong** [comment]
Sorry, forgot to deploy! Deployed now.

**Comment by gunar** [comment]
@michielbdejong it worked by then, but now it's broken again.

**Comment by gunar** [comment]
It's not completely broken. As @untitaker pointed out, if you click "update" the input-text field gets filled and the list of apps shown.

**Comment by raucao** [comment]
I think that's a UI design problem, not a bug.
